tests/cupy_tests/indexing_tests/test_generate.py

```diff
@@ -149,3 +149,41 @@ def test_AxisConcatenator_init1(self):
     def test_len(self):
         a = generate.AxisConcatenator()
         self.assertEqual(len(a), 0)
+
+
+@testing.gpu
+class TestUnravelIndex(unittest.TestCase):
+
+    _multiprocess_can_split_ = True
+
+    @testing.for_int_dtypes()
+    @testing.numpy_cupy_array_list_equal()
+    def test_C(self, xp, dtype):
+        a = testing.shaped_arange((4, 3, 2), xp, dtype)
+        a = xp.minimum(a, 6 * 4 - 1)
```

Comment:
Is this `minimum` needed?

Reply:
Without this `minimum`, `a` contains `24`. This index is invalid.

Reply:
OK.